start

```diff
@@ -78,7 +78,6 @@ function main() {
 
     # using stdbuf to eliminate any buffering between pipe,   
     stdbuf -o0 /start --$TARGET_NETWORK --enable-soroban-rpc $ARTIFICIAL_ACCELERATE --logs 2>&1 | stdbuf -i0 -o0 tee -a -i $LOG_FILE &
-    horizon_status
```

Comment:
since in there, if you don't mind minor clean-up, could remove the `else` block, `soroban_rpc_status` runs in any case.

Reply:
Done, PTAL